target/linux/ramips/dts/kn_st.dts

```diff
@@ -0,0 +1,114 @@
+/dts-v1/;
+
+#include "rt5350.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "zyxel,keenetic-start", "ralink,rt5350-soc";
+	model = "ZyXEL Keenetic Start";
+	
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+	};	
+	
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		led_status: power {
+			label = "zyxel:green:power";
+			gpios = <&gpio0 9 GPIO_ACTIVE_LOW>;
+		};
+
+		internet {
+			label = "zyxel:green:internet";
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+		};
+	};
+	
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 10 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 0 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {
```

Comment:
@SNMetamorph Can you please change the node name to more generic name `flash@0` and try if it boots for you?

Reply:
I applied this change, tested it on my router, and it boots and works.